PointOfDispensingApp/src/components/CreateLocationResourcePage.vue

```diff
@@ -0,0 +1,301 @@
+<template>
+  <v-container>
+    <v-row>
+      <h2 class="font-weight-medium primary--text">Location Resources</h2>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <v-divider></v-divider>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <div class="font-weight-medium secondary--text">Enter one or more search parameters to retrieve the available location resources.</div>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="5">
+        <v-row no-gutters>
+          <v-col cols="4">
+            <div class="secondary--text">City</div>
+          </v-col>
+          <v-col cols="8">
+            <v-text-field outlined dense 
+              v-model="search_city"
+            ></v-text-field>
+          </v-col>
+          <v-col cols="4">
+            <div class="secondary--text">Zip Code</div>
+          </v-col>
+          <v-col cols="8">
+            <v-text-field outlined dense 
+              v-model="search_postalCode"
+            ></v-text-field>
+          </v-col>
+        </v-row>
+      </v-col>
+      <v-col cols="1">
+        <!--blank column for spacing-->
+      </v-col>
+      <v-col cols="5">
+        <v-row no-gutters>
+          <v-col cols="4">
+            <div class="secondary--text">State</div>
+          </v-col>
+          <v-col cols="8">
+            <v-select
+              :items="stateOptions"
+              outlined
+              dense
+              v-model="search_state"
+            ></v-select>
+          </v-col>
+          <v-col cols="4">
+            <div class="secondary--text">Organization</div>
+          </v-col>
+          <v-col cols="8">
+            <v-text-field outlined dense 
+              v-model="search_organization"
+            ></v-text-field>
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row> 	
+    <v-row no-gutters>
+      <v-col cols="12">
+        <v-btn color="accent" @click="retrieveLocationResources()">
+          Search
+        </v-btn>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <v-divider></v-divider>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <template>
+          <v-data-table
+            @click:row="rowClick"
+            item-key="siteID"
+            single-select
+            :items="locationResourceLookupTable"
+            :headers="headers"
+            class="elevation-1"
+            :footer-props="{
+              'items-per-page-options':[5]
+            }"
+            :items-per-page="5"
+          ></v-data-table>
+        </template>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="12">
+        <v-dialog
```

Comment:
I'm not sure if this is normal UI/UX but you can't hit escape and close out of the dialog, which is a normal interaction for computer users. Is this something you're setting or no?

Reply:
Good catch David. I also noticed that you can't hit enter to "save" the entry either (you have to tab to get to save THEN hit enter). I know in this PR you don't actually save it but I think it makes sense to hit enter once you're on the zip code portion.

Reply:
I updated the code to allow the dialog to close when you hit Esc.  However, it doesn't look like Vue allows a dialog to be closed with the "Enter" key because the Enter key is used to activate various form elements (like drop-downs).  Would this be a desirable improvement?  If so, I'd suggest that we create a separate card for it.

Reply:
I'd say let's keep with only escaping with "esc" and not closing on "enter". Typically, you have to have selected the various entry in order to hit enter to move forward.